src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.activitieswithfruitservice.ActivitiesStrategy;
+import core.basesyntax.service.activitieswithfruitservice.impl.ActivitiesStrategyImpl;
+import core.basesyntax.service.readfromfileservice.ReadFromFileService;
+import core.basesyntax.service.readfromfileservice.impl.ReadFromFileCsvServiceImpl;
+import core.basesyntax.service.writetofileservice.WriteToFileService;
+import core.basesyntax.service.writetofileservice.impl.WriteToCsvFileServiceImpl;
+import java.util.List;
+
+public class Main {
+    private static final ReadFromFileService CSV_FILE_READER_SERVICE
+                = new ReadFromFileCsvServiceImpl();
+    private static final WriteToFileService CVS_FILE_WRITER_SERVICE
+            = new WriteToCsvFileServiceImpl();
+    private static final ActivitiesStrategy ACTIVITIES_STRATEGY = new ActivitiesStrategyImpl();
```

Comment:
Services are not immutable classes so even if they are static final they are not really contants so better to use regular naming case
```suggestion
    private static final ReadFromFileService csvFileReader
                = new ReadFromFileCsvServiceImpl();
    private static final WriteToFileService csvFileWriter
            = new WriteToCsvFileServiceImpl();
    private static final ActivitiesStrategy activitiesStrategy = new ActivitiesStrategyImpl();
```

Reply:
++